AngularSnapshotSerializer.js

```diff
@@ -29,6 +29,17 @@ const print = (val, print, indent, opts, colors) => {
     .map(node => Array.from(node.renderElement.childNodes).map(print).join(''))
     .join(opts.edgeSpacing);
 
+
+  const nodesWithoutDebugInformation = nodes
+    .split('\n')
```

Comment:
What if `opts.edgeSpacing` (see line 30) is not set to `\n`? I think this might not work anymore.

Better make your transformations before the html gets joined.

Reply:
Probably, I can just split and join lines by `opts.edgeSpacing`? What do you think?

Reply:
In line 30 the join is applied. If you use the nodes without joining them there you do not have to split them.

Your join afterwards should use `opts.edgeSpacing` then.